### ISSUE TITLE
Not running with last version of push plugin

### ISSUE THREAD

**Issue by jcus0006** [phonegap/phonegap-plugin-push#850]
I was using push plugin version 5.3. For some reason, when connecting the project to source control (tfs), the phonegap-plugin-push could no longer be found in the config.xml under plugins, even if the folder could still be found in the directory. In fact when the app tries to access the PushNotification object, it was not being found. So I tried to re-install the push plugin from the config.xml through github (and the latest version gets installed: with no choice). 

The problem is that when trying to run the app, errors are returned and it never installs on the device.

I am getting:
Output:

1>ERROR building one of the platforms : error : cmd: Command failed with exit code 1
1>                     You may not have the required environment or OS to build this project (TaskId:10)
1>16:26:21.820     1>
1>MDAVSCLI : error : cmd: Command failed with exit code 1
1>                   Done executing task "MdaVsCli" -- FAILED. (TaskId:10)
1>16:26:21.825     1>

Error List:
Command failed with exit code 1	ERROR building one of the platforms	1	
Command failed with exit code 1	MobileAppNew	C:\........\MobileAppNew\MobileAppNew\MDAVSCLI	1	

Running on LG Google Nexus 5 with latest Android 6.0.1

App runs properly with an older back up which was not connected to Team Foundation Server and was running with Push Plugin version 5.3.

Any help?

**Comment by macdonst** [comment]
@jcus0006 it seems like you can't compile the app and the important lines as to why it doesn't compile are above the error you listed. 

Also, you need to provide more details as per the issue submitting guidelines. It would be useful to know what version of cordova, cordova-android you are using, etc. which are asked for. As well the plugin hasn't gotten to release 5.3 so I'm not sure where you are getting that version number from.

**Comment by jcus0006** [comment]
Plugin version 5.3 means 1.5.3, that was my mistake. 

cordova -v returned 6.1.1

Installed platforms:
android 4.1.1
ios 3.9.2
windows 4.1.0

anything else I can provide, please ask. thanks for your help

**Comment by macdonst** [comment]
@jcus0006 did you get this sorted? As I said the actual error in the compilation shows up above the lines you pasted.

**Comment by jcus0006** [comment]
Hi, thank you. What I did that managed to fix this was update the Android SDK components from the ADB. I removed and re-added the Android platform. I also deleted the sub-folders of the platform folder in the cordova project and performed a clean build. I updated the cordova version to the latest one. Then I simply removed the plugin and re-installed it from github. One of these or a combination, should have solved the problem as now I am working with the latest versions. On a different note, I've reported another issue, this time with iOS in another post. Thank you for time again.
